### DISCUSSION TITLE
Welcome to big-sleep Discussions!

### DISCUSSION THREAD

**Discussion by lucidrains** [lucidrains/big-sleep#16] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by nerdyrodent** [comment]
Some random wonderings...
I love using music to drive GANs, so I was wondering how that could be done? At the moment I'm thinking iterations could be set to the song length, with each frame being changed slightly by the audio - or something along those lines?

What is the main difference between 2 epochs with 1k iterations, and 1 epoch with 2k iterations?
So far, I can see it does "change it's mind" sometimes. Some images seem to come out really well after 1 epoch and 400 iterations, and actually get worse later. I see a few with the characteristic "snake skin", like you get when you push BigGAN too far outside it's "comfort zone". Some just turn total white, or total black. Others will change drastically after about 6 epochs, though these seem to be more rare.

I was recently impressed by a GAN using sliced Wasserstein loss for style transfer. Not sure if this could be used to change the output quality?

This also really looks like it would work wonders as a style transfer tool... somehow. Using famous artists names seems to make very good images. "Pixel art" versions of things also seem to come out really well. Maybe something along the lines of the StyleGAN2 projection to match your supplied image first, then big-sleep on those latents?

I'm also guessing this BigGAN was trained on ImageNet? The first images it generates seems familiar ;)

**Comment by igouspoput** [comment]
hello, I just started github cuz I saw someone else use this big sleep, I have the desktop github thingy downloaded, and it said I needed "git bash" too. After that I downloaded the prompt and and extracted it and copied it to the desktop github thingy, and now I'm stuck, I have no idea what to do to be able to run the code, and I've tried looking at everything else. either I'm just blind and need help finding instructions, or someone needs to tell me how this all actually works, like do I need anything else? is there something that I downloaded that is actually stopping it instead of helping? someone just please help me
